scripts/git/pre-commit.linter_python

```diff
@@ -71,11 +71,11 @@ echo "  Analysing python code style with 'isort'."
 # could happen for example if the .isort.cfg src_paths list gets out
 # of date.
 
-if $(isort . --extend-skip cmake --profile black --diff --check-only --quiet >/dev/null); then
+if $(isort . --extend-skip cmake,submodules --profile black --diff --check-only --quiet >/dev/null); then
```

Comment:
Interesting. Should we propagate this change to the `isaac` repo? Not sure why the relevant commands are different.

https://github.com/nasa/isaac/blob/develop/scripts/git/pre-commit.linter_python#L75-L79

Reply:
(To answer my own question: "Yes." https://github.com/nasa/isaac/issues/123 )